### PR TITLE
Add human-metapneumovirus to supported pathogen repos

### PR DESCRIPTION
**Pull request by joverlee521**
## Description of proposed changes

As discussed in [PR review](https://github.com/nextstrain/infra/pull/34#discussion_r1882936107), the repo was renamed to `hmpv` to match the dataset names.

## Checklist

- [x] Checks pass

### CODE REVIEW

**Comment by joverlee521** [comment]
```
Terraform will perform the following actions:

  # aws_iam_policy.NextstrainPathogen["hmpv"] will be created
  + resource "aws_iam_policy" "NextstrainPathogen" {
      + arn         = (known after apply)
      + description = "Provides permissions to upload datasets, workflow files, etc. for a Nextstrain pathogen"
      + id          = (known after apply)
      + name        = "NextstrainPathogen@hmpv"
      + name_prefix = (known after apply)
      + path        = "/"
      + policy      = jsonencode(
            {
              + Statement = [
                  + {
                      + Action    = [
                          + "s3:ListBucket",
                          + "s3:ListBucketVersions",
                          + "s3:GetBucketLocation",
                          + "s3:GetBucketVersioning",
                        ]
                      + Condition = {
                          + StringLike = {
                              + "s3:prefix" = [
                                  + "hmpv.json",
                                  + "hmpv_*.json",
                                  + "files/workflows/hmpv/*",
                                  + "files/datasets/hmpv/*",
                                ]
                            }
                        }
                      + Effect    = "Allow"
                      + Resource  = [
                          + "arn:aws:s3:::nextstrain-data",
                          + "arn:aws:s3:::nextstrain-staging",
                        ]
                      + Sid       = "List"
                    },
                  + {
                      + Action    = [
                          + "s3:ListBucket",
                          + "s3:ListBucketVersions",
                          + "s3:GetBucketLocation",
                          + "s3:GetBucketVersioning",
                        ]
                      + Condition = {
                          + StringLike = {
                              + "s3:prefix" = [
                                  + "trial_*_hmpv.json",
                                  + "trial_*_hmpv_*.json",
                                ]
                            }
                        }
                      + Effect    = "Allow"
                      + Resource  = [
                          + "arn:aws:s3:::nextstrain-staging",
                        ]
                      + Sid       = "ListStagingTrials"
                    },
                  + {
                      + Action    = [
                          + "s3:ListBucket",
                          + "s3:ListBucketVersions",
                          + "s3:GetBucketLocation",
                          + "s3:GetBucketVersioning",
                        ]
                      + Condition = {
                          + StringLike = {
                              + "s3:prefix" = [
                                  + "files/workflows/hmpv/*",
                                  + "files/datasets/hmpv/*",
                                ]
                            }
                        }
                      + Effect    = "Allow"
                      + Resource  = [
                          + "arn:aws:s3:::nextstrain-data-private",
                        ]
                      + Sid       = "ListPrivate"
                    },
                  + {
                      + Action   = [
                          + "s3:GetObject",
                          + "s3:GetObjectTagging",
                          + "s3:GetObjectVersion",
                          + "s3:GetObjectVersionTagging",
                          + "s3:PutObject",
                          + "s3:PutObjectTagging",
                          + "s3:DeleteObject",
                        ]
                      + Effect   = "Allow"
                      + Resource = [
                          + "arn:aws:s3:::nextstrain-data/hmpv.json",
                          + "arn:aws:s3:::nextstrain-data/hmpv_*.json",
                          + "arn:aws:s3:::nextstrain-staging/hmpv.json",
                          + "arn:aws:s3:::nextstrain-staging/hmpv_*.json",
                          + "arn:aws:s3:::nextstrain-staging/trial_*_hmpv.json",
                          + "arn:aws:s3:::nextstrain-staging/trial_*_hmpv_*.json",
                          + "arn:aws:s3:::nextstrain-data/files/workflows/hmpv/*",
                          + "arn:aws:s3:::nextstrain-data/files/datasets/hmpv/*",
                          + "arn:aws:s3:::nextstrain-data-private/files/workflows/hmpv/*",
                          + "arn:aws:s3:::nextstrain-data-private/files/datasets/hmpv/*",
                          + "arn:aws:s3:::nextstrain-staging/files/workflows/hmpv/*",
                          + "arn:aws:s3:::nextstrain-staging/files/datasets/hmpv/*",
                        ]
                      + Sid      = "ReadWrite"
                    },
                  + {
                      + Action   = [
                          + "cloudfront:ListDistributions",
                        ]
                      + Effect   = "Allow"
                      + Resource = "*"
                      + Sid      = "CloudFrontList"
                    },
                  + {
                      + Action   = [
                          + "cloudfront:CreateInvalidation",
                          + "cloudfront:GetInvalidation",
                        ]
                      + Effect   = "Allow"
                      + Resource = [
                          + "arn:aws:cloudfront::827581582529:distribution/E3LB0EWZKCCV",
                          + "arn:aws:cloudfront::827581582529:distribution/E3L83FTHWUN0BV",
                        ]
                      + Sid      = "CloudFrontReadWrite"
                    },
                ]
              + Version   = "2012-10-17"
            }
        )
      + policy_id   = (known after apply)
      + tags_all    = (known after apply)
    }

  # aws_iam_role.GitHubActionsRoleNextstrainBatchJobs will be updated in-place
  ~ resource "aws_iam_role" "GitHubActionsRoleNextstrainBatchJobs" {
      ~ assume_role_policy    = jsonencode(
          ~ {
              ~ Statement = [
                  ~ {
                      ~ Condition = {
                          ~ StringLike = {
                              ~ "token.actions.githubusercontent.com:sub" = [
                                    # (4 unchanged elements hidden)
                                    "repo:nextstrain/forecasts-ncov:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*",
                                  + "repo:nextstrain/human-metapneumovirus:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*",
                                    "repo:nextstrain/lassa:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*",
                                    # (13 unchanged elements hidden)
                                ]
                                # (1 unchanged attribute hidden)
                            }
                        }
                        # (3 unchanged attributes hidden)
                    },
                ]
                # (1 unchanged attribute hidden)
            }
        )
        id                    = "GitHubActionsRoleNextstrainBatchJobs"
        name                  = "GitHubActionsRoleNextstrainBatchJobs"
        tags                  = {}
        # (11 unchanged attributes hidden)
    }

  # aws_iam_role.GitHubActionsRoleNextstrainRepo["human-metapneumovirus"] will be created
  + resource "aws_iam_role" "GitHubActionsRoleNextstrainRepo" {
      + arn                   = (known after apply)
      + assume_role_policy    = jsonencode(
            {
              + Statement = [
                  + {
                      + Action    = "sts:AssumeRoleWithWebIdentity"
                      + Condition = {
                          + StringLike = {
                              + "token.actions.githubusercontent.com:aud" = "sts.amazonaws.com"
                              + "token.actions.githubusercontent.com:sub" = "repo:nextstrain/human-metapneumovirus:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*"
                            }
                        }
                      + Effect    = "Allow"
                      + Principal = {
                          + Federated = "arn:aws:iam::827581582529:oidc-provider/token.actions.githubusercontent.com"
                        }
                    },
                ]
              + Version   = "2012-10-17"
            }
        )
      + create_date           = (known after apply)
      + description           = "Provides permissions to upload datasets, workflow files, etc. for a Nextstrain pathogen to select repos and select GitHub Actions OIDC workflows."
      + force_detach_policies = false
      + id                    = (known after apply)
      + managed_policy_arns   = (known after apply)
      + max_session_duration  = 43200
      + name                  = "GitHubActionsRoleNextstrainRepo@human-metapneumovirus"
      + name_prefix           = (known after apply)
      + path                  = "/"
      + role_last_used        = (known after apply)
      + tags_all              = (known after apply)
      + unique_id             = (known after apply)

      + inline_policy {
            name   = null
            # (1 unchanged attribute hidden)
        }
    }

  # github_actions_repository_oidc_subject_claim_customization_template.nextstrain["human-metapneumovirus"] will be created
  + resource "github_actions_repository_oidc_subject_claim_customization_template" "nextstrain" {
      + id                 = (known after apply)
      + include_claim_keys = [
          + "repo",
          + "context",
          + "job_workflow_ref",
        ]
      + repository         = "human-metapneumovirus"
      + use_default        = false
    }

  # github_repository_topics.pathogen["human-metapneumovirus"] will be created
  + resource "github_repository_topics" "pathogen" {
      + id         = (known after apply)
      + repository = "human-metapneumovirus"
      + topics     = [
          + "nextstrain",
          + "pathogen",
        ]
    }

Plan: 4 to add, 1 to change, 0 to destroy.
```

File: env/production/locals.tf [hunk 1]
```diff
@@ -14,6 +14,7 @@ locals {
     "avian-flu"       = ["avian-flu"],
     "dengue"          = ["dengue"],
     "forecasts-ncov"  = ["forecasts-ncov"],
+    "hmpv"            = ["human-metapneumovirus"],
```
Comment (tsibley) [inline env/production/locals.tf:17]: FWIW, I did think we had an (informal) policy of trying to avoid creating more mismatches like this.

Comment (victorlin) [reply env/production/locals.tf:17]: https://github.com/nextstrain/nextstrain.org/issues/41

Comment (genehack) [reply env/production/locals.tf:17]: Maybe the repo should be renamed?

Comment (joverlee521) [reply env/production/locals.tf:17]: [Asking on Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1734045738381789?thread_ts=1734039097.017049&cid=C01LCTT7JNN)

Comment (joverlee521) [reply env/production/locals.tf]: Repo has been renamed and updated repo name in https://github.com/nextstrain/infra/pull/34/commits/9cad7d4f1eee19da770b0b1ba19b0aafc8e354b3

**Comment by joverlee521** [comment]
```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create
  ~ update in-place

Terraform will perform the following actions:

  # aws_iam_policy.NextstrainPathogen["hmpv"] will be created
  + resource "aws_iam_policy" "NextstrainPathogen" {
      + arn         = (known after apply)
      + description = "Provides permissions to upload datasets, workflow files, etc. for a Nextstrain pathogen"
      + id          = (known after apply)
      + name        = "NextstrainPathogen@hmpv"
      + name_prefix = (known after apply)
      + path        = "/"
      + policy      = jsonencode(
            {
              + Statement = [
                  + {
                      + Action    = [
                          + "s3:ListBucket",
                          + "s3:ListBucketVersions",
                          + "s3:GetBucketLocation",
                          + "s3:GetBucketVersioning",
                        ]
                      + Condition = {
                          + StringLike = {
                              + "s3:prefix" = [
                                  + "hmpv.json",
                                  + "hmpv_*.json",
                                  + "files/workflows/hmpv/*",
                                  + "files/datasets/hmpv/*",
                                ]
                            }
                        }
                      + Effect    = "Allow"
                      + Resource  = [
                          + "arn:aws:s3:::nextstrain-data",
                          + "arn:aws:s3:::nextstrain-staging",
                        ]
                      + Sid       = "List"
                    },
                  + {
                      + Action    = [
                          + "s3:ListBucket",
                          + "s3:ListBucketVersions",
                          + "s3:GetBucketLocation",
                          + "s3:GetBucketVersioning",
                        ]
                      + Condition = {
                          + StringLike = {
                              + "s3:prefix" = [
                                  + "trial_*_hmpv.json",
                                  + "trial_*_hmpv_*.json",
                                ]
                            }
                        }
                      + Effect    = "Allow"
                      + Resource  = [
                          + "arn:aws:s3:::nextstrain-staging",
                        ]
                      + Sid       = "ListStagingTrials"
                    },
                  + {
                      + Action    = [
                          + "s3:ListBucket",
                          + "s3:ListBucketVersions",
                          + "s3:GetBucketLocation",
                          + "s3:GetBucketVersioning",
                        ]
                      + Condition = {
                          + StringLike = {
                              + "s3:prefix" = [
                                  + "files/workflows/hmpv/*",
                                  + "files/datasets/hmpv/*",
                                ]
                            }
                        }
                      + Effect    = "Allow"
                      + Resource  = [
                          + "arn:aws:s3:::nextstrain-data-private",
                        ]
                      + Sid       = "ListPrivate"
                    },
                  + {
                      + Action   = [
                          + "s3:GetObject",
                          + "s3:GetObjectTagging",
                          + "s3:GetObjectVersion",
                          + "s3:GetObjectVersionTagging",
                          + "s3:PutObject",
                          + "s3:PutObjectTagging",
                          + "s3:DeleteObject",
                        ]
                      + Effect   = "Allow"
                      + Resource = [
                          + "arn:aws:s3:::nextstrain-data/hmpv.json",
                          + "arn:aws:s3:::nextstrain-data/hmpv_*.json",
                          + "arn:aws:s3:::nextstrain-staging/hmpv.json",
                          + "arn:aws:s3:::nextstrain-staging/hmpv_*.json",
                          + "arn:aws:s3:::nextstrain-staging/trial_*_hmpv.json",
                          + "arn:aws:s3:::nextstrain-staging/trial_*_hmpv_*.json",
                          + "arn:aws:s3:::nextstrain-data/files/workflows/hmpv/*",
                          + "arn:aws:s3:::nextstrain-data/files/datasets/hmpv/*",
                          + "arn:aws:s3:::nextstrain-data-private/files/workflows/hmpv/*",
                          + "arn:aws:s3:::nextstrain-data-private/files/datasets/hmpv/*",
                          + "arn:aws:s3:::nextstrain-staging/files/workflows/hmpv/*",
                          + "arn:aws:s3:::nextstrain-staging/files/datasets/hmpv/*",
                        ]
                      + Sid      = "ReadWrite"
                    },
                  + {
                      + Action   = [
                          + "cloudfront:ListDistributions",
                        ]
                      + Effect   = "Allow"
                      + Resource = "*"
                      + Sid      = "CloudFrontList"
                    },
                  + {
                      + Action   = [
                          + "cloudfront:CreateInvalidation",
                          + "cloudfront:GetInvalidation",
                        ]
                      + Effect   = "Allow"
                      + Resource = [
                          + "arn:aws:cloudfront::827581582529:distribution/E3LB0EWZKCCV",
                          + "arn:aws:cloudfront::827581582529:distribution/E3L83FTHWUN0BV",
                        ]
                      + Sid      = "CloudFrontReadWrite"
                    },
                ]
              + Version   = "2012-10-17"
            }
        )
      + policy_id   = (known after apply)
      + tags_all    = (known after apply)
    }

  # aws_iam_role.GitHubActionsRoleNextstrainBatchJobs will be updated in-place
  ~ resource "aws_iam_role" "GitHubActionsRoleNextstrainBatchJobs" {
      ~ assume_role_policy    = jsonencode(
          ~ {
              ~ Statement = [
                  ~ {
                      ~ Condition = {
                          ~ StringLike = {
                              ~ "token.actions.githubusercontent.com:sub" = [
                                    # (4 unchanged elements hidden)
                                    "repo:nextstrain/forecasts-ncov:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*",
                                  + "repo:nextstrain/hmpv:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*",
                                    "repo:nextstrain/lassa:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*",
                                    # (13 unchanged elements hidden)
                                ]
                                # (1 unchanged attribute hidden)
                            }
                        }
                        # (3 unchanged attributes hidden)
                    },
                ]
                # (1 unchanged attribute hidden)
            }
        )
        id                    = "GitHubActionsRoleNextstrainBatchJobs"
        name                  = "GitHubActionsRoleNextstrainBatchJobs"
        tags                  = {}
        # (11 unchanged attributes hidden)
    }

  # aws_iam_role.GitHubActionsRoleNextstrainRepo["hmpv"] will be created
  + resource "aws_iam_role" "GitHubActionsRoleNextstrainRepo" {
      + arn                   = (known after apply)
      + assume_role_policy    = jsonencode(
            {
              + Statement = [
                  + {
                      + Action    = "sts:AssumeRoleWithWebIdentity"
                      + Condition = {
                          + StringLike = {
                              + "token.actions.githubusercontent.com:aud" = "sts.amazonaws.com"
                              + "token.actions.githubusercontent.com:sub" = "repo:nextstrain/hmpv:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*"
                            }
                        }
                      + Effect    = "Allow"
                      + Principal = {
                          + Federated = "arn:aws:iam::827581582529:oidc-provider/token.actions.githubusercontent.com"
                        }
                    },
                ]
              + Version   = "2012-10-17"
            }
        )
      + create_date           = (known after apply)
      + description           = "Provides permissions to upload datasets, workflow files, etc. for a Nextstrain pathogen to select repos and select GitHub Actions OIDC workflows."
      + force_detach_policies = false
      + id                    = (known after apply)
      + managed_policy_arns   = (known after apply)
      + max_session_duration  = 43200
      + name                  = "GitHubActionsRoleNextstrainRepo@hmpv"
      + name_prefix           = (known after apply)
      + path                  = "/"
      + role_last_used        = (known after apply)
      + tags_all              = (known after apply)
      + unique_id             = (known after apply)

      + inline_policy {
            name   = null
            # (1 unchanged attribute hidden)
        }
    }

  # github_actions_repository_oidc_subject_claim_customization_template.nextstrain["hmpv"] will be created
  + resource "github_actions_repository_oidc_subject_claim_customization_template" "nextstrain" {
      + id                 = (known after apply)
      + include_claim_keys = [
          + "repo",
          + "context",
          + "job_workflow_ref",
        ]
      + repository         = "hmpv"
      + use_default        = false
    }

  # github_repository_topics.pathogen["hmpv"] will be created
  + resource "github_repository_topics" "pathogen" {
      + id         = (known after apply)
      + repository = "hmpv"
      + topics     = [
          + "nextstrain",
          + "pathogen",
        ]
    }

Plan: 4 to add, 1 to change, 0 to destroy.
```

**Comment by joverlee521** [comment]
Changes applied

```
github_actions_repository_oidc_subject_claim_customization_template.nextstrain["hmpv"]: Creating...
github_repository_topics.pathogen["hmpv"]: Creating...
aws_iam_policy.NextstrainPathogen["hmpv"]: Creating...
aws_iam_role.GitHubActionsRoleNextstrainBatchJobs: Modifying... [id=GitHubActionsRoleNextstrainBatchJobs]
aws_iam_policy.NextstrainPathogen["hmpv"]: Creation complete after 0s [id=arn:aws:iam::827581582529:policy/NextstrainPathogen@hmpv]
aws_iam_role.GitHubActionsRoleNextstrainRepo["hmpv"]: Creating...
aws_iam_role.GitHubActionsRoleNextstrainBatchJobs: Modifications complete after 0s [id=GitHubActionsRoleNextstrainBatchJobs]
aws_iam_role.GitHubActionsRoleNextstrainRepo["hmpv"]: Creation complete after 1s [id=GitHubActionsRoleNextstrainRepo@hmpv]
github_actions_repository_oidc_subject_claim_customization_template.nextstrain["hmpv"]: Creation complete after 3s [id=hmpv]
github_repository_topics.pathogen["hmpv"]: Creation complete after 3s [id=hmpv]

Apply complete! Resources: 4 added, 1 changed, 0 destroyed.
```

**Pull request merged**